.pylintrc

```diff
@@ -3,7 +3,7 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
-extension-pkg-allow-list=
+extension-pkg-allow-list=pydantic.dataclasses
```

Comment:
I had to add `lxml` to this as well, but that depends on your C++ apigen implementation.